lib/mocha/mocha.ts

```diff
@@ -170,4 +170,4 @@
 
   })(Mocha.Runner.prototype.runTest, Mocha.Runner.prototype.run);
 
-})(window);
\ No newline at end of file
+})(typeof window !== 'undefined' ? window : global);
```

Comment:
please use 
```javascript
})(typeof window !== 'undefined' && window || typeof self !== 'undefined' && self || global);
```

which is used here https://github.com/angular/zone.js/blob/master/lib/zone.ts#L1344

Reply:
done